code/modules/antagonists/voidwalker/voidwalker_bodyparts.dm

```diff
@@ -0,0 +1,159 @@
+/obj/item/bodypart/head/voidwalker
+	texture_bodypart_overlay = /datum/bodypart_overlay/texture/spacey
+	icon_greyscale = 'icons/mob/human/species/voidwalker.dmi'
+	limb_id = SPECIES_VOIDWALKER
+	is_dimorphic = FALSE
+	bodypart_traits = list(TRAIT_MUTE)
+	head_flags = NONE
+	blocks_emissive = EMISSIVE_BLOCK_NONE
+
+	brute_modifier = 0.9
+	burn_modifier = 0.8
+
+	damage_overlay_color = list(1, 2, 2, 0,
+								0, 1, 0, 0,
```

Comment:
Where are you going why are you all the way over there
if you want all the numbers to be aligned just press enter before `1, 2, 2, 0`